apollo-router/src/plugins/connectors/http_json_transport.rs

```diff
@@ -77,43 +78,74 @@ pub(crate) fn make_request(
         &flat_inputs,
     )?;
 
-    let (json_body, body, apply_to_errors) = if let Some(ref selection) = transport.body {
+    let request = http::Request::builder()
+        .method(transport.method.as_str())
+        .uri(uri.as_str());
+
+    // add the headers and if content-type is specified, we'll check that when constructing the body
+    let (mut request, content_type) = add_headers(
+        request,
+        original_request.supergraph_request.headers(),
+        &transport.headers,
+        &flat_inputs,
+    );
+
+    let is_form_urlencoded = content_type.as_ref() == Some(&mime::APPLICATION_WWW_FORM_URLENCODED);
```

Comment:
`CONTENT_TYPE` is still in `RESERVED_HEADERS` up above. What impact does that have elsewhere?

Reply:
that's used for `{ name: "x" from: "y" }` — it won't forward the content type from the graphql client. that does actually mean i can remove a tiny bit of code — good catch!

Reply:
right now it'll emit a warning log, so disallowing might be the right idea. let's file a ticket

Reply:
https://apollographql.atlassian.net/browse/CNN-413